js/arena.js

```diff
@@ -0,0 +1,33 @@
+import Gladiators from './gladiator.js'
+
+export default class Arena {
+    constructor(name, gladiators = [],) {
+        this.name = name.charAt(0).toUpperCase() + name.slice(1)
+        this.gladiators = gladiators;
+    }
+    addGladiator(name) {
+        if (this.gladiators.length < 2){
+            this.gladiators.push(name);
+        } else {
+        return `Too many Gladiators in the Arena!`
```

Comment:
In order to show the message in the console used this
console.log('Too many Gladiators in the Arena!');